syntax="proto3";
package settings;

message GetSettings {
  bool enable_emg = 1;
  bool enable_display = 2;
  bool enable_gyro = 3;
  bool enable_driver = 4;
}

message SetSettings {
  bool enable_emg = 1;
  bool enable_display = 2;
  bool enable_gyro = 3;
  bool enable_driver = 4;
  bool power_off = 5;
}
